modules/cats-effect/src/main/scala/scalacache/CatsEffect.scala

```diff
@@ -1,20 +1,15 @@
 package scalacache
 
-import cats.effect.{Async => CatsAsync, IO}
+import cats.Functor
+import cats.evidence.<~<
+import cats.effect.{Async => CatsAsync, IO, Resource}
 
 import scala.language.higherKinds
 import scala.util.control.NonFatal
 
 object CatsEffect {
 
-  object modes {
-
-    /**
-      * A mode that wraps computations in cats-effect IO.
-      */
-    implicit val io: Mode[IO] = new Mode[IO] {
-      val M: Async[IO] = asyncForCatsEffectAsync[IO]
-    }
+  object lowPrioModes {
```

Comment:
I'm a bit confused. Why do you need to split this into 2 objects? This would be a breaking change for anyone who is using cats-effect `Async` but not `IO`.

Reply:
oh yeah you're right. I was intending to make this a proper lowPrio trait but screwed it up. I had ambigous implicit issues for `Async[IO]` due to both the explicit instance `io` and the def `async` matching. Why are those both in there and not just the `async` one since that should cover cats `IO` as well?

Reply:
Explanation of why both `IO` and `Async` are there: https://github.com/cb372/scalacache/pull/173#discussion_r182006753

But the `IO` one could probably be removed without anyone noticing, if it's causing you problems.